### PR TITLE
Update FAQ

### PR DESCRIPTION
**Pull request by kian-thompson**

### CODE REVIEW

**Review by anthony-murphy — APPROVED** [review]
looks good to me. lets let @tylerbutler  chime in before commit

**Comment by tylerbutler** [comment]
The current audience for fluidframework.com isn't going to use data stores directly or interact with channels, so I'm a little reluctant to add this to the FAQ without more work to expand our docs to cover the entire framework. @ssimic2 has been thinking about the overall API docs in #8847 and I started some early thinking about a new section on the site in #8323.

**Comment by anthony-murphy** [comment]
> The current audience for fluidframework.com isn't going to use data stores directly or interact with channels, so I'm a little reluctant to add this to the FAQ without more work to expand our docs to cover the entire framework. @ssimic2 has been thinking about the overall API docs in #8847 and I started some early thinking about a new section on the site in #8323.

maybe put this in wiki instead: https://github.com/microsoft/FluidFramework/wiki

overall i'm really confused on the right place to put docs. i really want a place we can cheaply add content, as if the bar it too high we don't add anything.

File: docs/content/docs/faq.md [hunk 1]
```diff
@@ -62,6 +62,24 @@ one Fluid service.
 Fluid clients connect to the Fluid service using the WebSocket protocol. However, the Fluid runtime manages
 all of the connections so that Fluid client developers can focus on local experiences.
 
+### What are data stores used for?
+
+Data stores are used to collect DDSes under a single parent. A DDS must be parented by to a data store and you can treat data stores to be collections of DDSes.
+
+### What is the difference between data stores and data objects?
+
+In core FluidFramework we have data stores. As the name suggests, it is a place where you can store data. It contains one or more DDSs that have the actual user data. A data store is responsible (among other things) for serializing this data to be stored (in a file) and then re-hydrating this data when the file is loaded.
```
Comment (scarlettjlee) [inline docs/content/docs/faq.md:71]: @tylerbutler, isn't "DDSs" supposed to cause a build break?

**Comment by tylerbutler** [comment]
@kian-thompson did the right thing here -- write the docs _somewhere_ then let us argue about where they _should_ be. :)

File: docs/content/docs/faq.md (hunk 1 above)
Comment (tylerbutler) [reply docs/content/docs/faq.md:71]: It should! Will file an issue.

Comment (tylerbutler) [reply docs/content/docs/faq.md:71]: #9978

**Pull request closed without merging**